SBAT.md

```diff
@@ -453,7 +453,7 @@ And the UEFI CA issues an update to SBAT which has:
 sbat,1
 shim,1
 grub,3
-grub.fedora,1
+grub.fedora,2
 ```
```

Comment:
Right, the doc also says:

```
At this point, each vendor updates their grub builds, and updates the component_generation in .sbat to 1
```
and that also should be `2`. Could you please also fix that while being there ?